build.py

```diff
@@ -33,6 +33,8 @@ def run(cmd: str, **kwargs) -> subprocess.CompletedProcess:
 def build():
     run(f"{PYTHON_EXECUTABLE_PATH} --version")
     run(f"{PYTHON_EXECUTABLE_PATH} misc/generate_pyqt.py")
+    if sys.platform == "linux":
```

Comment:
Why ? We need a comment to understand - what is achieved here and why it is needed !

Reply:
Also we don't need that for macOS ?

Reply:
1. `patchelf --version` print the version of `patchelf` is for debug purpose, to be sure it use the correct version (not the system one)
2. `elf` is linux specific or darwin it's `mach-o` file